### PR TITLE
[cmake] flatbuffer build test off

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds EXTRA_OPTS to Flatbuffers build for resolving gcc bugs

Related : #3390 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
Q) this change only affects flatbuffers internal test codes?

**Comment by seanshpark** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by mhs4670go** [comment]
@seanshpark Right:)

File: infra/cmake/packages/FlatBuffersConfig.cmake [hunk 1]
```diff
@@ -26,6 +26,7 @@ function(_FlatBuffers_build)
                       INSTALL_DIR ${EXT_OVERLAY_DIR}
                       BUILD_FLAGS ${ADDITIONAL_CXX_FLAGS}
                       IDENTIFIER  "1.10-fix1"
```
Comment (hseok-oh) [inline infra/cmake/packages/FlatBuffersConfig.cmake:28]: Please update `-fix1` -> `-fix2` for rebuild.

Comment (hseok-oh) [inline infra/cmake/packages/FlatBuffersConfig.cmake:28]: Please update `-fix1` -> `-fix2` for rebuild.

**Pull request merged**